test-d/entrypoint-node.test-d.ts

```diff
@@ -25,6 +25,8 @@ import {
   After,
   BeforeStep,
   AfterStep,
+  BeforeAll,
+  AfterAll,
```

Comment:
You can add the same tests here, as you have in `test-d/entrypoint-browser.test-d.ts`: